GW2EIEvtcParser/EIData/CombatReplay/CombatReplayDecorationContainer.cs

```diff
@@ -306,7 +306,7 @@ internal void AddTetherByEffectGUID(ParsedEvtcLog log, EffectEvent effect, Color
     /// <param name="firstAwareThreshold">Time threshold in case the agent spawns before the buff application.</param>
     internal void AddTetherByThirdPartySrcBuff(ParsedEvtcLog log, PlayerActor player, long buffId, int buffSrcAgentId, int toTetherAgentId, string color, int firstAwareThreshold = 2000)
     {
-        var buffEvents = log.CombatData.GetBuffDataByIDByDst(buffId, player.AgentItem).Where(x => x.CreditedBy.IsSpecies(buffSrcAgentId));
+        var buffEvents = log.CombatData.GetBuffDataByIDByDst(buffId, player.AgentItem).Where(x => x.CreditedBy.IsSpecies(buffSrcAgentId)).ToList();
```

Comment:
Why the change?

Reply:
To not iterate the enumerable twice underneath, if you don't deem it necessary I'll remove it

Reply:
I'm not sure creating a list + iterating over twice is better then iterating over an enumerable twice. Particularly in this scope where where we will rarely get more than a few dozens elements in the enumerable. I think you can rollback this.